ServerCode/application/AppServer.py

```diff
@@ -2,38 +2,21 @@
 This module provides functionalities to upload files.
 Convert JSON to CSV and allow dowloading the CSV.
 """
-import logging
-import logging.config
+import sys
 import os
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+import logging
 import webbrowser
-import sys
-
-# Third-party imports
-from flask import Flask, jsonify, redirect, request, send_from_directory, url_for
-from flask_cors import CORS
 from werkzeug.utils import secure_filename
+from flask import Flask, request, jsonify, send_from_directory, redirect, url_for
+from flask_cors import CORS
+from BubbleScan_AI.Scantron import Scantron95945
```

Comment:
@ramezmosad I think the problem @Chirag2x reported has to do with this import. Try using this instead:
```suggestion
from application.Scantron import Scantron95945
```
As you mentioned, there are two copies of Scantron.py, and I think we accidentally switched to using the wrong copy.

Reply:
I tried this solution earlier but it's still not working. The issue might be something else.